pypsa/optimization/global_constraints.py

```diff
@@ -477,13 +481,27 @@ def substr(s):
             )
 
             if not isnan(period):
-                ext_i = ext_i[n.get_active_assets(c, period)[ext_i]].rename(ext_i.name)
+                ext_i = ext_i[n.get_active_assets(c, period)[ext_i]].rename(
+                    ext_i.name
+                )  # or extendable only ?
```

Comment:
what do you mean here?

Reply:
That's some scrap of a question I asked myself but solved since, and forgot to remove ^^'